packages/webapp/src/components/Modals/QuickAssignModal/index.jsx

```diff
@@ -36,13 +68,34 @@ export default function TaskQuickAssignModal({
     } else return [selfOption, unAssignedOption];
   }, []);
 
-  const [selectedWorker, setWorker] = useState(isAssigned ? unAssignedOption : selfOption);
-  const [assignAll, setAssignAll] = useState(false);
+  const [showHourlyWageSection, setShowHourlyWageSection] = useState(false);
+  const [showHourlyWageInput, setShowHourlyWageInput] = useState(false);
+
+  const {
+    control,
+    register,
+    resetField,
+    watch,
+    formState: { isValid, errors },
+  } = useForm({
+    mode: 'onTouched',
+    defaultValues: {
+      [ASSIGNEE]: isAssigned ? unAssignedOption : selfOption,
+      [HOURLY_WAGE_ACTION]: '',
+      [HOURLY_WAGE]: null,
+      [ASSIGN_ALL]: false,
+    },
+  });
+
+  const selectedWorker = watch(ASSIGNEE);
+  const selectedHourlyWageAction = watch(HOURLY_WAGE_ACTION);
+  const hourlyWage = watch(HOURLY_WAGE);
+  const assignAll = watch(ASSIGN_ALL);
 
   const tasks = useSelector(tasksSelector);
+  const currencySymbol = grabCurrencySymbol(getCurrencyFromStore());
```

Comment:
getCurrencyFromStore is deprecated.
It was used as walk around in some old class component.